scripts/bundler.sh

```diff
@@ -70,6 +75,9 @@ main() {
       ELECTRON_URL="https://github.com/electron/electron/releases/download/$electron_version/$ELECTRON"
       download_jre_and_electron
 
+      PLAYWRIGHT_PLATFORM="mac"
```

Comment:
add cpu architecture to the platform

Reply:
It has nothing to do with cpu architecture, it only has to do with the location of the download on Playwrights CDN